pkg/cloudprovider/providers/azure/azure_util_test.go

```diff
@@ -51,3 +51,31 @@ func TestGetScaleSetVMInstanceID(t *testing.T) {
 		}
 	}
 }
+
+func TestGetStorageAccountName(t *testing.T) {
+	tests := []struct {
+		prefix   string
+		uniqueID string
+	}{
+		{
+			prefix:   "",
+			uniqueID: "",
+		},
+		{
+			prefix:   "pvc",
+			uniqueID: "resourcegroupwestuscc1624c7-3f1d-4ed3-a855-668a86e96ad8",
+		},
+		{
+			prefix:   "1234512345123451234512345",
+			uniqueID: "resourcegroupwestuscc1624c7-3f1d-4ed3-a855-668a86e96ad8",
+		},
+	}
+
+	for _, test := range tests {
+		accountName := getStorageAccountName(test.prefix, test.uniqueID)
+		// fmt.Printf("input prefix: %s, uniqueID: %s, output account name: %s\n", test.prefix, test.uniqueID, accountName)
```

Comment:
Remove debug info. Also can assert account name length >= 3.
`Storage account names must be between 3 and 24 characters `

Reply:
fixed